usagov/vote-gov-nvrf-app
# VOTE-916: Implement field wrapper component

Jira ticket:
[VOTE-916](https://cm-jira.usa.gov/browse/VOTE-916)

Testing:
1. inside of /PersonalInfo.jsx add this line to imports:
`import FieldBuilder from '../FieldBuilder.jsx';`
`import FieldsetContainer from '../FieldsetContainer.jsx';`
2. and this block inside of the return
`        <FieldContainer 
            fieldType={'text'} 
            inputData={{id: 'first_name', required: 1, label: 'this is a label', error_msg: 'this is an error message', help_text: 'this is help text'}}
            saveFieldData={props.saveFieldData}
            fieldData={props.fieldData}
        />`
`        <FieldContainer  fieldType={'select'}  inputData={{id: 'title', required: 1, label: 'this is a label', error_msg: 'this is an error message', help_text: 'this is help text', options: [{key: 'Mr.', value: 'Mr'}, {key: 'Miss', value: 'Miss'}, {key: 'Ms.', value: 'Ms'}, {key: 'Mrs.', value: 'Mrs'}]}} saveFieldData={props.saveFieldData} fieldData={props.fieldData} />`
`        <FieldsetContainer  fieldType={'date'}  inputData={{id: 'date-of-birth', required: 1, label: 'this is a label', error_msg: 'this is an error message', help_text: 'this is help text'}} saveFieldData={props.saveFieldData} dateFormat={props.dateFormat} checkDateValues={checkDateValues} fieldData={props.fieldData} />`
3. run `npm run start` and navigate to personal info page
4. check that field renders
5. test usability and error handling
